src/reporter/command/command-formatter.ts

```diff
@@ -125,6 +129,10 @@ export class CommandFormatter {
         return command.url;
     }
 
+    private _prepareCustomActionResult (): unknown {
```

Comment:
You can remove this function since it's used only in one place and contains only one instruction.